### PR TITLE
Removing dangerous unwrap() calls within the library in non-test envs

### LINKED ISSUE #139: Remove calls to `unwrap()`
We need to strip the repo of any panicking calls, such as `unwrap`. This may also include changing return types for various messages.

### PR DESCRIPTION
**Pull request by slawlor**
In the library, there was a lot of use of ```unwrap()``` in dangerous places. This PR cleans this up

Resolves #139

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Installation
 Add the following line to the dependencies of your `Cargo.toml`:
 
 ```
-akd = "0.3"
+akd = "0.4"
```
Comment (eozturk1) [inline README.md:26]: Have we bumped the version to 0.4?

File: akd_mysql/src/mysql.rs [hunk 2]
```diff
@@ -952,82 +960,92 @@ impl Storage for AsyncMySqlDatabase {
...
-                                    DbRecord::get_multi_row_specific_params::<St>(batch).unwrap(),
-                                );
-                            }
-                        }
-
-                        // insert the batches of size = MYSQL_EXTENDED_INSERT_DEPTH
-                        if !params.is_empty() {
-                            let fill_statement = DbRecord::get_batch_fill_temp_table::<St>(Some(
-                                self.tunable_insert_depth,
+                let results = if let Some(create_table_cmd) =
```
Comment (eozturk1) [inline akd_mysql/src/mysql.rs:963]: Is it possible to rewrite this with an early else for better readability? i.e., 

```
if else_condition {
  return None;
}
// Proceed with valid results value
```

**Review by eozturk1 — APPROVED** [review]
LGTM. Added a comment about the `akd` version bump in the README and a readability question.

File: README.md (hunk 1 above)
Comment (slawlor) [reply README.md:26]: Yes due to some unsightly contract changes between lib's, it was necessary to get the CI to not complain.

File: akd_mysql/src/mysql.rs (hunk 2 above)
Comment (slawlor) [reply akd_mysql/src/mysql.rs:963]: Not really a great way, we'd lose timing information with an early exit (debug! prompts).

**Pull request merged**